annotationProcessor/build.gradle

```diff
@@ -39,9 +39,9 @@ dependencies {
         implementation 'com.nimbusds:nimbus-jose-jwt:9.37.3'
     }
 
-    implementation "org.slf4j:slf4j-api:${gradle.vSlf4jApi}"
```

Comment:
Build script variables do not need gradle qualifier, hence removed in all references